atqa-framework/.eslintrc.json

```diff
@@ -0,0 +1,70 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint",
+    "prettier",
+    "import",
+    "codeceptjs"
+  ],
+  "env": {
+    "node": true,
+    "codeceptjs/codeceptjs": true
+  },
+  "extends": [
+    "airbnb-typescript/base",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier",
+    "prettier/@typescript-eslint",
+    "plugin:codeceptjs/recommended"
+  ],
+  "parserOptions": {
+    "project": ["tsconfig.json"]
+  },
+  "rules": {
+    "prettier/prettier": ["error", { "singleQuote": true }],
+    "no-var": "error",
```

Comment:
Included in `plugin:@typescript-eslint/recommended`

Reply:
what u mean included? you should specify rules for plugin in the rules section

Reply:
https://github.com/wavesplatform/waves-keeper/pull/408#discussion_r953409713